docs/source/conf.py

```diff
@@ -0,0 +1,57 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+sys.path.insert(0, os.path.abspath('../..'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'sphinxext-opengraph'
+copyright = '2020, FIRST'
+author = 'WPILib'
+
+# The full version, including alpha/beta/rc tags
+release = '1.0'
```

Comment:
Shouldn't this match the version of the package?

Reply:
Doesn't matter too much, it only shows up in the PDF version of the docs, which will be disabled